libminifi/test/TestBase.cpp

```diff
@@ -18,6 +18,21 @@
 
 #include "./TestBase.h"
 
+#include "spdlog/spdlog.h"
+
+void LogTestController::setLevel(const std::string name, spdlog::level::level_enum level) {
+  logger_->log_info("Setting log level for %s to %s", name, spdlog::level::to_str(level));
+  std::string adjusted_name = name;
+  const std::string clazz = "class ";
+  auto haz_clazz = name.find(clazz);
+  if (haz_clazz == 0)
+    adjusted_name = name.substr(clazz.length(), name.length() - clazz.length());
+  if (config && config->shortenClassNames()) {
+    utils::ClassUtils::shortenClassName(adjusted_name, adjusted_name);
+  }
```

Comment:
This has just been moved from the h to the cpp file to avoid some include issues. You are welcome to create a refactor task for this, if it is used in more places I would agree that a good solution.

Reply:
I didn't realize that it's only been moved. Nevermind then.